io_u.c

```diff
@@ -742,6 +742,30 @@ static enum fio_ddir rate_ddir(struct thread_data *td, enum fio_ddir ddir)
 	return ddir;
 }
 
+static inline enum fio_ddir get_sync_ddir(const struct thread_data *td)
+{
+	if (should_fsync(td))
+	{
+		if (td->o.fsync_blocks && td->io_issues[DDIR_WRITE] &&
+			!(td->io_issues[DDIR_WRITE] % td->o.fsync_blocks))
```

Comment:
Should align with above, again observe surrounding style and match it. Same for the rest of this function.